app/routes.php

```diff
@@ -188,4 +189,58 @@ function( Request $request ) use ( $app, $ffFactory ) {
 	}
 );
 
+$app->post(
+	'donation/add',
+	function( Application $app, Request $request ) use ( $ffFactory ) {
+		$useCase = $ffFactory->newAddDonationUseCase();
+
+		$donationRequest = new DonationRequest();
+		$donationRequest->setAmount( floatval( $request->get( 'betrag', 0 ) ) );
```

Comment:
This clashes with the German locale, where amounts like "5,99" will be parsed to "5.00". Not sure where the locale-specific stuff belongs, we'll have to discuss that with @JeroenDeDauw